Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -20,13 +20,20 @@ enum class SettingType
   Bool,
 };
 
+enum class SettingDifficulty
```

Comment:
'Difficulty' seems like a strange way to phrase this, though I'm not quite sure of a better name for this either...

Reply:
How about `SettingVisibility`?

Reply:
That works, yeah.

Reply:
Are you going to make this change? I would just merge this other than this little thing.

Reply:
Yes. In a few hours I should get to it.